Added documentation of indexes for case-insensitive databases (#4209)
wso2/docs-is: en/identity-server/next/docs/deploy/configure/databases/carbon-database/change-to-mssql.md

    2.  Execute database scripts.
    
        Execute the scripts in the `<IS-HOME>/dbscripts/mssql.sql` file, against the database created.

> !!! note

Author: resolved with https://github.com/wso2/docs-is/pull/4209/commits/24ec8d148e9a0a8fa9efc843b02d9856a987e914

>     If Microsoft SQL Server is used since they are case-insensitive by default (subject to confirmation with the current collation of the DB), 

Author: resolved with https://github.com/wso2/docs-is/pull/4209/commits/24ec8d148e9a0a8fa9efc843b02d9856a987e914

        Disabling this feature in {{product_name}} eliminates unnecessary processing, thereby potentially enhancing overall system performance.

> 

Reviewer: ```suggestion

```

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> 

Author: resolved with https://github.com/wso2/docs-is/pull/4209/commits/24ec8d148e9a0a8fa9efc843b02d9856a987e914

>         To disable this functionality, open the `deployment.toml` file found in the `<IS-HOME>/repository/conf/` directory and add the following configurations to the primary user store.

Reviewer: ```suggestion
        To disable case-insensitivity for the primary user store, open the `deployment.toml` file found in the `<IS-HOME>/repository/conf/` directory and add the following configurations to the primary user store.
```

    2.  Execute database scripts.
    
        Execute the scripts in the `<IS-HOME>/dbscripts/mssql.sql` file, against the database created.

    !!! note

>         When integrating WSO2 Identity Server with Microsoft SQL Server, it is advisable to deactivate the case-insensitive functionality within WSO2 IS.

Reviewer: ```suggestion
        As Microsoft SQL Server is inherently case-insensitive, it is recommended to deactivate the case-insensitive functionality of {{product_name}} when integrating with the Microsoft SQL Server.
```

>         This recommendation is based on the inherent case-insensitivity of Microsoft SQL Server, which renders the similar feature in WSO2 IS redundant.

Reviewer: ```suggestion
```

>         Disabling this feature in WSO2 IS eliminates unnecessary processing, thereby potentially enhancing overall system performance.

Reviewer: ```suggestion
        Disabling this feature in {{product_name}} eliminates unnecessary processing, thereby potentially enhancing overall system performance.
```

>         For the primary user store, add the following configurations to the `<IS-HOME>/repository/conf/deployment.` file.

Reviewer: ```suggestion
        To disable this functionality, open the `deployment.toml` file found in the `<IS-HOME>/repository/conf/` directory and add the following configurations to the primary user store.
```

        ``` toml
        [user_store.properties]
        CaseInsensitiveUsername = false
        UseCaseSensitiveUsernameForCacheKeys = false
        ```  
    

>         For secondary user stores, add the following configurations to the `<userstore>.xml` file in the `<IS_HOME>/repository/deployment/server/userstores` folder.

Reviewer: ```suggestion
        For secondary user stores, add the following configurations to the `<userstore>.xml` file found in the `<IS_HOME>/repository/deployment/server/userstores` directory.
```
